dev-packages/browser-integration-tests/suites/tracing/linking-addLink/test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from '@playwright/test';
+import type { Event, SpanJSON } from '@sentry/core';
+import { sentryTest } from '../../../utils/fixtures';
+import { getMultipleSentryEnvelopeRequests, shouldSkipTracingTest } from '../../../utils/helpers';
+
+sentryTest('should link spans with addLink() in trace context', async ({ getLocalTestUrl, page }) => {
+  if (shouldSkipTracingTest()) {
+    sentryTest.skip();
+  }
+
+  const url = await getLocalTestUrl({ testDir: __dirname });
+  const [rootSpan1, rootSpan2] = await getMultipleSentryEnvelopeRequests<Event>(page, 3, { url });
```

Comment:
if we need to wait for 3 events here (do we?) but only want 2, my gut feeling tells me that there's room for flakes. Let's use the `waitForTransactionRequest` helper instead and check for the correct `transaction` (or some other identifier). This should be safer for flakes and makes things more explicit. 

(side note: I think we should generally use this helper more often, as it works out really well in our e2e tests. But obviously, no action required, we can do this iteratively and over time)